nipype/interfaces/ants/preprocess.py

```diff
@@ -0,0 +1,358 @@
+                      "{sampling_percentage}]")
+        format_args = {}
+        metric_args = ["metric_type", "fixed_image", "moving_image",
+                       "metric_weight", "radius_or_bins", "sampling_strategy",
+                       "sampling_percentage"]
+        for metric_arg in metric_args:
+            if _extant(getattr(self.inputs, metric_arg)):
+                format_args[metric_arg] = getattr(self.inputs, metric_arg)
+        return metric_str.format(**format_args)
+
+    def _format_transform(self):
+        transform_str = "-t {}[{}]"
+        if (_extant(self.inputs.transformation_model)
+                and _extant(self.inputs.gradient_step_length)):
+            return transform_str.format(self.inputs.transformation_model,
+                                        self.inputs.gradient_step_length)
+        raise ValueError("Unable to format transformation_model argument")
+
+    def _format_output(self):
+        if (_extant(self.inputs.output_transform_prefix)
+                and _extant(self.inputs.output_warped_image)
+                and _extant(self.inputs.output_average_image)):
+            return "-o [{},{},{}]".format(
+                self.inputs.output_transform_prefix,
+                self.inputs.output_warped_image,
+                self.inputs.output_average_image
+            )
+        elif _extant(self.inputs.output_average_image):
+            return "-o {}".format(self.inputs.output_average_image)
+        else:
+            raise ValueError("Unable to format output due to lack of inputs.")
+
+    def _list_outputs(self):
+        outputs = self._outputs().get()
+        if _extant(self.inputs.output_average_image):
+            outputs['average_image'] = (
+                os.path.abspath(self.inputs.output_average_image)
+            )
+        if _extant(self.inputs.output_warped_image):
+            outputs['warped_image'] = (
+                os.path.abspath(self.inputs.output_warped_image)
+            )
+        if _extant(self.inputs.output_transform_prefix):
+            fname = '{}MOCOparams.csv'.format(
+                self.inputs.output_transform_prefix
+            )
+            outputs['composite_transform'] = os.path.abspath(fname)
+        if (_extant(self.inputs.write_displacement) and
+                _extant(self.inputs.output_transform_prefix) and
+                self.inputs.write_displacement is True):
+            fname = '{}Warp.nii.gz'.format(self.inputs.output_transform_prefix)
+            outputs['displacement_field'] = os.path.abspath(fname)
+        return outputs
+
+class AntsMatrixConversionInputSpec(BaseInterfaceInputSpec):
+    matrix = File(
+        exists=True,
+        desc='Motion crrection matrices to be converted into FSL style motion parameters',
+        mandatory=True
+    )
+
+class AntsMatrixConversionOutputSpec(TraitedSpec):
+    parameters = File(exists=True, desc="parameters to be output")
+
+
+class AntsMatrixConversion(BaseInterface):
```

Comment:
I'd be more specific here: something like `MotionParameters2FSL` instead of `AntsMatrixConversion`